docs/source/configuration/locking.md

```diff
@@ -0,0 +1,21 @@
+# Locking support
+
+Volto provides support for Plone's locking feature, but you need to enable the `plone.locking` behavior on your Dexterity content types first. You can do that in Plone's control panel or using the Generic Setup facilities.
+
+## Volto configuration
+
+You need to enable locking support in Volto's configuration object:
+
+```js
+import config from '@plone/volto/registry'
+
+config.settings.hasLockingSupport = true;
```

Comment:
Could you elaborate why we do not enable this feature by default? If I am not mistaken, locking is always enabled in Plone if the behavior is active.

Reply:
No strong reason. Guillotina, maybe?! Old plone.restapi :thinking: To be discussed.